paimon-core/src/main/java/org/apache/paimon/migrate/FileMetaUtils.java

```diff
@@ -60,6 +61,20 @@ public class FileMetaUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(FileMetaUtils.class);
 
+    public static List<DataFileMeta> construct(
+            List<IcebergDataFileMeta> icebergDataFileMetas,
+            FileIO fileIO,
+            Table paimonTable,
+            Path newDir,
+            Map<Path, Path> rollback) {
```

Comment:
This method is only related to Iceberg, while other methods in this utility class are quite versatile. Move this method to `IcebergMigrator`.